- name: slsAPINetworksPOSTEntryBadData - attempt to create an entry with bad data
  request:
    url: "{sls_base_url}/v1/networks"
    method: POST
    verify: !bool "{verify}"
    json: { "foo": "dummy","Type": "ethernet", "IPRanges":["127.0.0.1/8"]}
  response:
    status_code: 400
    verify_response_with:
      function: tavern.testutils.helpers:validate_pykwalify
      extra_kwargs:
        schema:
          type: map
          required: True
          mapping:
            type:
              type: str
              required: True
            title:
              type: str
              required: True
            detail:
              type: str
              required: True
            instance:
              type: str
              required: True
            status:
              type: int
              required: True
